ios/magnet/AppDelegate.m

```diff
@@ -52,8 +54,17 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
   UIViewController *rootViewController = [UIViewController new];
   rootViewController.view = rootView;
   self.window.rootViewController = rootViewController;
+  [NotificationsHelper register];
   [self.window makeKeyAndVisible];
   return YES;
 }
 
+- (void)applicationDidBecomeActive:(UIApplication *)application {
+  [NotificationsHelper clearNotifications];
+}
+
+- (void)applicationDidEnterBackground:(UIApplication *)application {
+  [NotificationsHelper clearNotifications];
```

Comment:
I don't want to mess with the timer stuff, as it can get tricky, but in the didEnterBackground we could do something like:
`[NotificationHelper disableNotifications]'

and then in the didBecomeActive:
`[NotificationHelper enableNotifications]'